trs_filer/api/service-info.yaml

```diff
@@ -0,0 +1,122 @@
+openapi: 3.0.2
```

Comment:
Please use the same naming scheme with date and commit hash for the service info specs